xoinvader/animation.py

```diff
@@ -252,11 +254,11 @@ def interpolate(first, second, current_time):
     if not first[0] <= current_time <= second[0]:
         raise AnimationBoundariesExceeded(first[0], current_time, second[0])
 
-    def frames_of(types):
+    def frames_of(*args):
         """If frames both of specified type."""
-        return same_type((first[1], second[1]), types)
+        return same_type((first[1], second[1]), args)
```

Comment:
It's ok to pass list to `same_type`?
If yes, then `same_type`'s docstring needs an update too.